certexchange/polling/metrics.go

```diff
@@ -0,0 +1,62 @@
+package polling
+
+import (
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+var meter = otel.Meter("f3/certexchange/polling")
+var metrics = struct {
+	activePeers              metric.Int64Gauge
+	backoffPeers             metric.Int64Gauge
+	predictedPollingInterval metric.Float64Gauge
+	pollDuration             metric.Float64Histogram
+	peersPolled              metric.Int64Histogram
+	peersRequiredPerPoll     metric.Int64Histogram
+	pollEfficiency           metric.Float64Histogram
+}{
+	activePeers: must(meter.Int64Gauge(
+		"f3_certexchange_active_peers",
```

Comment:
Include "polling" or "poll" in prefix to avoid possible naming conflict with metrics at parent package?